.prettierignore

```diff
@@ -1,6 +1,7 @@
 .github/**/*.md
 dist
 test/unit/coverage
+test/tapes
```

Comment:
We can add them to .dockerignore as well?

Reply:
We already have the `/test` directory in `.dockerignore` already. That does mean all nested folders, doesn't it, @sarayourfriend ?